### ISSUE TITLE
Implement fragment

### ISSUE THREAD

**Issue by ClassicOldSong** [TheNeuronProject/ef.js#12]
For now a component can only be created within a tag, thus is not really convenient for some complicated circumstances. For example, a template like:
```
>h1
  .Hello, World!
>h2
  .Hello, ef.js
```
could be rendered as:
```
<h1>Hello, World!</h1>
<h2>Hello, ef.js</h2>
```

This can also be helpful when using jsx implementation(#3), while React provides `React.fragment` which does exactly the same thing.

**Comment by ClassicOldSong** [comment]
Mounting points in fragments is not working properly, will have a fix in 0.9.1

**Comment by ClassicOldSong** [comment]
Fixed in v0.9.1
